samtranslator/model/sam_resources.py

```diff
@@ -2149,7 +2156,7 @@ class SamGraphQLApi(SamResourceMacro):
         "SchemaUri": Property(False, IS_STR),
     }
 
-    Auth: Auth
+    Auth: Dict[str, Any]
```

Comment:
Why? Is the value `Any`?

Reply:
@aahung brought up a good point when I was talking with him. Typing nested SAM properties (dictionaries) is not great because we haven't verified at this step what is actually in the dictionary, so it can do more harm than good to type it as such. So I am just using a very broad dictionary typing instead.

The CloudFormation resource typings are fine to use because we have a bunch of validation before hand to confirm they are of appropriate shape, so those will remain.